MdeModulePkg/Library/MemoryBucketLib/MemoryBucketLib.inf

```diff
@@ -0,0 +1,36 @@
+## @file
+# Library the defines Runtime memory buckets for use during PEI

```

Comment:
This library doesn't defined the buckets, right? It provides logic to manage the buckets in PEI.

Reply:
Updated.